modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/get_lexicon_icon.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default function getLexiconIcon(icon, cssClass = '') {
+	if (!icon) {
+		throw new TypeError('Parameter icon must be provided');
+	}
+
+	function getLexiconIconTemplate(iconName, cssClass) {
+		return `<svg
+				aria-hidden="true"
+				class="lexicon-icon lexicon-icon-${iconName} ${cssClass}"
+				focusable="false"
+				role="presentation"
+			>
+				<use href="${themeDisplay.getPathThemeImages()}/clay/icons.svg#${iconName}" />
+			</svg>`;
+	}
+
+	const lexiconIconTemplate = getLexiconIconTemplate(icon, cssClass);
+	const tempElement = document.createElement('div');
+
+	tempElement.innerHTML = lexiconIconTemplate;
+
+	return tempElement.firstChild;
```

Comment:
Should we use a fragment here? Just a thought, I don't think it makes much difference in this case.